os_stub/spdm_device_secret_lib_sample/lib.c

```diff
@@ -1597,7 +1597,7 @@ bool libspdm_write_certificate_to_nvm(uint8_t slot_id, const void * cert_chain,
     uint64_t fp_out;
 #endif
 
-    char file_name[] = {'s','l','o','t','_','i','d','_','0','\0'};
+    char file_name[] = "slot_id_0";
```

Comment:
please change to "slot_id_0_cert_chain.der".

Reply:
Thanks. I have changed the file name.